src/v/kafka/client/test/fixture.h

```diff
@@ -28,6 +28,7 @@ class kafka_client_fixture : public redpanda_thread_fixture {
         }).get0();
         app.initialize(proxy_config(), proxy_client_config());
         app.check_environment();
+        app.clear_migrators();
```

Comment:
This is related to https://github.com/redpanda-data/redpanda/pull/6835/commits/794ee987fcca9123b992e0dc0177aeb4561ec2dc#r1007360272

Reply:
This is before the migrators are constructed in wire_up_and_start, so won't do the trick.